### PR TITLE
Adds formula for wifi-password

### PR DESCRIPTION
**Pull request by jeduan**
Added formula for wifi-password (https://github.com/rauchg/wifi-password)

### CODE REVIEW

**Comment by jeduan** [comment]
ref: https://github.com/rauchg/wifi-password/issues/4

File: Library/Formula/wifi-password.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class WifiPassword < Formula
+  homepage "https://github.com/rauchg/wifi-password"
+  url "https://github.com/rauchg/wifi-password/archive/0.0.1.tar.gz"
+  version "0.0.1"
```
Comment (tdsmith) [inline Library/Formula/wifi-password.rb]: `version` is redundant with the version extracted from the tarball filename so you can remove it

File: Library/Formula/wifi-password.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  sha1 "7f92794a3a04762009a0c4525cbf2aa851002289"
+
+  def install
+    bin.install "wifi-password.sh" => "wifi-password"
+  end
+
+  def test
```
Comment (tdsmith) [inline Library/Formula/wifi-password.rb]: use `test do`

**Comment by tdsmith** [comment]
wifi-password --version doesn't match the tarball's release version, does it?

File: Library/Formula/wifi-password.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Formula/wifi-password.rb]: `0.1.0` has been released now too.

File: Library/Formula/wifi-password.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    system "#{bin}/wifi-password", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/wifi-password.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by dmyers** [comment]
:+1:

File: Library/Formula/wifi-password.rb (hunk 3 above)
Comment (jeduan) [inline Library/Formula/wifi-password.rb]: Trying to think of a test that can be run even when the host computer isn't connected to a wireless network, but can't think of anything. Any ideas?

**Comment by jeduan** [comment]
Updated to reflect v0.1.0

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `wifi-password 0.1.0 (new formula)`.

**Pull request closed without merging**